### DISCUSSION TITLE
Issue with usage of CLI options

### DISCUSSION THREAD

**Discussion by ruanye** [jplag/JPlag#1890] [Q&A] [answered]
I use the new version, the error is as follows, help how to solve
my  tree    
```                                                   9:06:12
.
├── jplag.jar
└── test
    ├── 15535142
    │   └── 01
    │       └── 1.js
    ├── 15537806
    │   └── 01
    │       └── 1.js
    └── 15539271
        └── 01
            └── 1.js
```

[aa] java -jar jplag.jar -l javascript -s ./test -r ./test-result      9:06:14

2024-07-25-09:06:19_966 [INFO] CLI - Summary of all errors:
2024-07-25-09:06:19_965 [ERROR] CLI - No root directories specified with submissions to check for plagiarism!

**Comment by tsaglam** [comment, marked as answer]
You are using the legacy CLI syntax, use `-h` to show the CLI help text.

Drop the `-s` and try `java -jar jplag.jar -l javascript ./test -r test-result`.
